### PR TITLE
Noting what the returned Value can contain 

### PR DESCRIPTION
**Pull request by JackTrapper**
## Summary

It's tricky because columns that are Rid or CodedTokens were implicitly converted to full tokens.

And also noting that for String, Guid, and Blob columns, the returned value is a token that must be passed to a corresponding heap reading function (GetString, GetGuid, GetBlob).

And also note that a column that contains, for example, an 8-bit value is automatically up-converted to the full 32-bit `pVal` return value.

### CODE REVIEW

**Comment by mairaw** [comment]
@noahfalk can you please review this one as well?

File: docs/framework/unmanaged-api/metadata/imetadatatables-getcolumn-method.md
Comment (mairaw) [inline docs/framework/unmanaged-api/metadata/imetadatatables-getcolumn-method.md]:
```suggestion
The interpretion of the value returned through `pVal` depends on the column's type. The column type can be determined by calling [IMetaDataTables.GetColumnInfo](imetadatatables-getcolumninfo-method.md).
```

File: docs/framework/unmanaged-api/metadata/imetadatatables-getcolumn-method.md
Comment (mairaw) [inline docs/framework/unmanaged-api/metadata/imetadatatables-getcolumn-method.md]:
```suggestion
- The `GetColumn` method automatically converts columns of type **Rid** or **CodedToken** to full 32-bit `mdToken` values.
```

File: docs/framework/unmanaged-api/metadata/imetadatatables-getcolumn-method.md
Comment (mairaw) [inline docs/framework/unmanaged-api/metadata/imetadatatables-getcolumn-method.md]: we avoid the use of future tense when possible. 

```suggestion
- For *heap* type columns, the returned *pVal* is an index into the corresponding heap.
```

File: docs/framework/unmanaged-api/metadata/imetadatatables-getcolumn-method.md
Comment (mairaw) [inline docs/framework/unmanaged-api/metadata/imetadatatables-getcolumn-method.md]:
```suggestion
| `iCodedToken`..`iCodedTokenMax`<br>(64..95) | mdToken | Upon return, *pVal* will contain a full Token. The function automatically decompresses the CodedToken into a full token. |
```

File: docs/framework/unmanaged-api/metadata/imetadatatables-getcolumn-method.md
Comment (mairaw) [inline docs/framework/unmanaged-api/metadata/imetadatatables-getcolumn-method.md]:
```suggestion
| `iSHORT` (96)            | Int16         | Automatically sign-extended to 32-bit.  |
```

File: docs/framework/unmanaged-api/metadata/imetadatatables-getcolumn-method.md
Comment (mairaw) [inline docs/framework/unmanaged-api/metadata/imetadatatables-getcolumn-method.md]:
```suggestion
| `iUSHORT` (97)           | UInt16        | Automatically sign-extended to 32-bit.  |
```

File: docs/framework/unmanaged-api/metadata/imetadatatables-getcolumn-method.md
Comment (mairaw) [inline docs/framework/unmanaged-api/metadata/imetadatatables-getcolumn-method.md]:
```suggestion
| `iBYTE` (100)            | Byte          | Automatically sign-extended to 32-bit.  |
```

File: docs/framework/unmanaged-api/metadata/imetadatatables-getcolumn-method.md
Comment (mairaw) [inline docs/framework/unmanaged-api/metadata/imetadatatables-getcolumn-method.md]:
```suggestion
| `iSTRING` (101)          | String heap index | *pVal* is an index into the String heap. Use [IMetadataTables.GetString](imetadatatables-getstring-method.md) to get the actual column String value. |
```

File: docs/framework/unmanaged-api/metadata/imetadatatables-getcolumn-method.md
Comment (mairaw) [inline docs/framework/unmanaged-api/metadata/imetadatatables-getcolumn-method.md]:
```suggestion
| `iGUID` (102)            | GUID heap index | *pVal* is an index into the GUID heap. Use [IMetadataTables.GetGuid](imetadatatables-getguid-method.md) to get the actual column GUID value. |
```

File: docs/framework/unmanaged-api/metadata/imetadatatables-getcolumn-method.md
Comment (mairaw) [inline docs/framework/unmanaged-api/metadata/imetadatatables-getcolumn-method.md]:
```suggestion
| `iBLOB` (103)            | Blob heap index | *pVal* is an index into the Blob heap. Use [IMetadataTables.GetBlob](imetadatatables-getblob-method.md) to get the actual column Blob value. |
```

File: docs/framework/unmanaged-api/metadata/imetadatatables-getcolumn-method.md
Comment (mairaw) [inline docs/framework/unmanaged-api/metadata/imetadatatables-getcolumn-method.md]: also if you can:
line 3 - change ms.date to today's date
line 35 - remove two hashes from the parameters heading

**Review by mairaw — COMMENTED** [review]
Thanks @JackTrapper. Left a few grammar suggestions while we wait for Noah's technical review.

**Review by noahfalk — APPROVED** [review]

**Comment by Thraka** [comment]
@JackTrapper PING

**Comment by JackTrapper** [comment]
I assume i'm not doing something right. Every few weeks it seems i have to come back and hit **Accept**, **Save**, **Commit**, **Resolve**, **Check-in**. 

I wrote new text, and wanted to save it in the wiki. But it seems that it keeps coming back to me; as if i'm somehow an arbiter of something.

I just wanted to help make the docs better; and GitHub won't accept them...

**Comment by Thraka** [comment]
@JackTrapper Hi Jack, sorry for bothering you 😄 And thank you for taking the time to do some edits. I don't want you to think that we're not grateful!!

So GitHub doesn't work like a wiki. Your GitHub account contains your changes and with a pull request you ask for those changes to be incorporated into the master document that is hosted by our GitHub account. 

In reviewing the pull request, we can suggest clarifications and adjustments. We can actually edit in the PR and "suggest" a commit to be applied by you. We cannot edit the content owned by your account directly. So the GitHub suggestion mechanism allows us to suggest the change, but you as the owner of your copy of the content in GitHub, have to accept that change. The commit button on the suggestion does the accept, the resolve conversation button essentially ignores it (if you didn't want the change or were going to address it in a different way).

@mairaw I think we should just accept this PR as it is and circle back and implement the suggested changes ourselves or through a community issue. What do you think?

**Comment by mairaw** [comment]
We can also edit the PR ourselves @Thraka. We just prefer that the person who opened it addresses the issue, but if it's taking a long time, we can also edit.

**Comment by mairaw** [comment]
I've applied the remaining feedback and I'll merge once the build succeeds.

**Review by mairaw — APPROVED** [review]

**Pull request merged**
